src/Umbraco.Core/Services/UserService.cs

```diff
@@ -869,6 +869,11 @@ private UserOperationStatus ValidateUserCreateModel(UserCreateModel model)
             return UserOperationStatus.InvalidEmail;
         }
 
+        if (model.Id is not null && await GetAsync(model.Id.Value) is null)
```

Comment:
```suggestion
        if (model.Id is not null && await GetAsync(model.Id.Value) is not null)
```
I can't create new users 😢 was this meant to be `is not null`?